app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/onboarding/OnboardingNavigation.kt

```diff
@@ -0,0 +1,40 @@
+package com.x8bit.bitwarden.ui.auth.feature.onboarding
+
+import androidx.navigation.NavController
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.NavHostController
+import androidx.navigation.NavOptions
+import androidx.navigation.navigation
+import com.x8bit.bitwarden.ui.auth.feature.accountsetup.SETUP_UNLOCK_ROUTE
+import com.x8bit.bitwarden.ui.auth.feature.accountsetup.setupUnlockDestination
+
+const val ONBOARDING_GRAPH_ROUTE: String = "onboarding_graph"
```

Comment:
Wanna tos a kdoc on here

Reply:
I can, I don't see that for the other public route constants?